navigation/AppNavigator.js

```diff
@@ -31,11 +32,11 @@ function DrawerNavigator() {
         component={StoresStackNavigator}
         options={{ title: 'Stores' }}
       />
-      {/* <Drawer.Screen
+      <Drawer.Screen
         name="Rewards"
         options={{ title: 'Points History', drawerLockMode: 'locked-closed' }}>
         {props => <RewardsScreen {...props} tab={1} />}
-      </Drawer.Screen> */}
+      </Drawer.Screen>
```

Comment:
This was supposed to be left out for Guest Mode since there are no tabs in Reward for Guest Mode. We should take it out again.